helpers/fail2ban

```diff
@@ -86,28 +86,43 @@ port = __PORTS__
 filter = __APP__
 logpath = __LOGPATH__
 maxretry = __MAX_RETRY__
-" >$YNH_APP_BASEDIR/conf/f2b_jail.conf
+" >"$YNH_APP_BASEDIR/conf/f2b_jail.conf"
 
         echo "
 [INCLUDES]
 before = common.conf
 [Definition]
 failregex = __FAILREGEX__
 ignoreregex =
-" >$YNH_APP_BASEDIR/conf/f2b_filter.conf
+" >"$YNH_APP_BASEDIR/conf/f2b_filter.conf"
     fi
 
     ynh_add_config --template="f2b_jail.conf" --destination="/etc/fail2ban/jail.d/$app.conf"
     ynh_add_config --template="f2b_filter.conf" --destination="/etc/fail2ban/filter.d/$app.conf"
 
+    # if "$logpath" doesn't exist (as if using --use_template argument), assign
+    # "$logpath" using the one in the package's conf/f2b_jail.conf template
+    if [ -z "$logpath" ]; then
+        # the first sed deletes possibles spaces and the second one extract the path
+        logpath=$(grep logpath "$YNH_APP_BASEDIR/conf/f2b_jail.conf" | sed "s/ //g" | sed "s/logpath=//g")
+        # replace any '__VAR__' by their real variable using ynh_replace_vars
+        # in some hacky way because ynh_replace_vars supports only files ^^'
+        tempdir="$(mktemp -d)"
+        echo "$logpath" > "$tempdir/process-logpath"
+        ynh_replace_vars --file="$tempdir/process-logpath"
+        logpath="$(cat "$tempdir/process-logpath")"
+        rm -r "$tempdir"
```

Comment:
In fact I realize that we can probably `grep` directly the destination file, which is rendered right before with the vars and stuff ?

```suggestion
        logpath=$(grep logpath "/etc/fail2ban/jail.d/$app.conf" | sed "s/ //g" | sed "s/logpath=//g")
```

Reply:
hm yes but we should use ynh_replace_vars (or any equivalent) to get a proper logpath, otherwise the path used to `touch` will be incorrect :/

Reply:
maybe something like this?

```bash
tempdir="$(mktemp -d)"
grep logpath "$YNH_APP_BASEDIR/conf/f2b_jail.conf" | sed "s/ //g" | sed "s/logpath=//g" > "$tempdir/process-logpath"
ynh_replace_vars --file="$tempdir/process-logpath"
logpath="$(cat "$tempdir/process-logpath")"
rm -r "$tempdir"
```

Reply:
Eeeh but the point is that `/etc/fail2ban/jail.d/$app.conf` is generated with `ynh_add_config` which already does "hydrate" the template with the proper `__FOOBAR__` stuff replaced